components/calendar/src/cal/chinese.rs

```diff
@@ -73,13 +59,24 @@ mod simple;
 /// * _The Mathematics of the Chinese Calendar_ by Helmer Aslaksen <https://citeseerx.ist.psu.edu/viewdoc/download?doi=10.1.1.139.9311&rep=rep1&type=pdf>
 /// * Wikipedia: <https://en.wikipedia.org/wiki/Chinese_calendar>
 ///
-/// # Month codes
+/// # Months and days
+///
+/// The 12 months are called Zōuyuè (`M01`), Xìngyuè (`M02`), Táoyuè (`M03`),
+/// Méiyuè (`M04`), Liúyuè (`M05`), Héyuè (`M06`), Lányuè (`M07`), Guìyuè (`M08`),
+/// Júyuè (`M09`), Lùyuè (`M10`), Dōngyuè (`M11`), Bīngyuè (`M12`).
+///
+/// As a lunar calendar, the lengths of the months depend on the lunar cycle (a month starts on the day of
+/// local new moon), and will be either 29 or 30 days. As the lunar and siderial month do not line up, a leap
+/// month is inserted roughly every 3 years; this can be after any month (e.g. `M02L`).
+///
+/// Both the lengths of the months and the occurence of leap months are determined by the
+/// concrete [`Rules`] implementation.
+///
+/// The length of the year is 353-355 days, and the length of the leap year 383-385 days.
 ///
-/// This calendar is a lunisolar calendar. It supports regular month codes `"M01" - "M12"` as well
-/// as leap month codes `"M01L" - "M12L"`.
+/// # Siderial drift
 ///
-/// This calendar is currently in a preview state: formatting for this calendar is not
-/// going to be perfect.
+/// As leap months are determined with respect to the siderial year, this calendar does not experience siderial drift.
```

Comment:
issue: this is incorrect. the leap months are determined with respect to the tropical year. We do not have any sidereal calendars in ICU4X (Hindu is the only one that works that way).

"siderial drift" is unfortunately an ambiguous term since it sometimes covers both tropical drift and actual siderial drift. I would recommend avoiding it, because a calendar must have either tropical or sidereal drift, they are incompatible goals.